superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -364,7 +364,7 @@ function ChartList(props: ChartListProps) {
       Header: t('Search'),
       id: 'slice_name',
       input: 'search',
-      operator: 'name_or_description',
+      operator: 'chart_all_text',
```

Comment:
Looks like arg_names share a global namespace? I got a test failure on the previously added `all_text` filter for saved queries. If so, we should add the the resource name to other custom filters so there's no confusion/collision.

Reply:
Yes, good point!